content/technotes/2017-11-08-solrium-solr-r.md

```diff
@@ -0,0 +1,278 @@
+
+```r
+SolrClient$new(host = "foobar.com", path = "search", port = NULL)
+```
+
+If the instance uses SSL, simply specify that like:
+
+```r
+SolrClient$new(host = "foobar.com", path = "search", port = NULL, scheme = "https")
+```
+
+<br>
+
+## Query and body parameters
+
+Another big change in the package is that we wanted to make it easy to determine whether your Solr query gets passed as query parameters in a `GET` request or as body in a `POST` request. Solr clients in some other languages do this, and it made sense to port over that idea here. Now you pass your key-value pairs to either `params` or `body`. If nothing is passed to `body`, we do a `GET` request. If something is passed to `body` we do a `POST` request, even if there's also key-value pairs passed to `params`.
+
+This change does break the interface we had in the old version, but we think it's worth it.
+
+For example, to do a search you have to pass the collection name and a list of named parameters:
+
+
+```r
+conn$search(name = "gettingstarted", params = list(q = "*:*"))
+```
+
+```
+#> # A tibble: 5 x 5
+#>      id   title title_str  `_version_` price
+#>   <chr>   <chr>     <chr>        <dbl> <int>
+#> 1    10 adfadsf   adfadsf 1.582913e+18    NA
+#> 2    12  though    though 1.582913e+18    NA
+#> 3    14 animals   animals 1.582913e+18    NA
+#> 4     1    <NA>      <NA> 1.582913e+18   100
+#> 5     2    <NA>      <NA> 1.582913e+18   500
+```
+
+You can instead pass the connection object to `solr_search`:
+
+
+```r
+solr_search(conn, name = "gettingstarted", params = list(q = "*:*"))
+```
+
+```
+#> # A tibble: 5 x 5
+#>      id   title title_str  `_version_` price
+#>   <chr>   <chr>     <chr>        <dbl> <int>
+#> 1    10 adfadsf   adfadsf 1.582913e+18    NA
+#> 2    12  though    though 1.582913e+18    NA
+#> 3    14 animals   animals 1.582913e+18    NA
+#> 4     1    <NA>      <NA> 1.582913e+18   100
+#> 5     2    <NA>      <NA> 1.582913e+18   500
+```
+
+And the same pattern applies for the other functions:
+
+- `solr_facet`
+- `solr_group`
+- `solr_mlt`
+- `solr_highlight`
+- `solr_stats`
+- `solr_all`
+
+<br>
+
+## New functions for atomic updates
+
+[A user requested](https://github.com/ropensci/solrium/issues/97) the ability to do [atomic updates](https://lucene.apache.org/solr/guide/7_0/updating-parts-of-documents.html) - partial updates to documents without having to re-index the entire document.
+
+Two functions were added: `update_atomic_json` and `update_atomic_xml` for JSON and XML based updates. Check out their help pages for usage.
+
+<br>
+
+## Search results as attributes
+
+`solr_search` and `solr_all` in `v1` gain attributes that include `numFound`, `start`, and `maxScore`. That is, you can get to these three values after data is returned. Note that some Solr instances may not return all three values.
+
+For example, let's use the Public Library of Science Solr search instance:
```

Comment:
link to http://api.plos.org/? 
I think it's valuable at this point since you're using it in example